tools/api-markdown-documenter/src/test/snapshots/markdown/simple-suite-test/deep-config/test-suite-a/testbetanamespace-namespace/publicmember-variable.md

```diff
@@ -0,0 +1,11 @@
+# publicMember
+
+[Packages](/) &gt; [test-suite-a](/test-suite-a/) &gt; [TestBetaNamespace](/test-suite-a/testbetanamespace-namespace/) &gt; [publicMember](/test-suite-a/testbetanamespace-namespace/publicmember-variable)
+
+**WARNING: This API is provided as a beta preview and may change without notice. Use at your own risk.**
```

Comment:
Note that this item gets the beta warning, despite being tagged as `@public`, because its parent namespace is tagged as `@beta`.